mitodl/bootcamp-ecommerce
# feat(import_wire_transfers)!: use bootcamp run id in import_wire_transfers

#### What are the relevant tickets?
Closes https://github.com/mitodl/hq/issues/3632

#### What's this PR do?
Updates the `import_wire_transfers` management command to support the Bootcamp run ID instead of the Name and start date in the CSV.

Name and start date can be present in the CSV but we won't use them. We will just use the Bootcamp Run ID to import the wire transfers.

#### How should this be manually tested?

- Create a sample CSV with these fields. `Id,Learner Email,Amount,Bootcamp Run ID,Bootcamp Start Date,Bootcamp Name`
- Add some sample records.
- Make sure to add the Bootcamp Run ID
- Use `import_wire_transfers` to import the CSV
- It should be successful.
